gluoncv/data/pascal_voc/detection.py

```diff
@@ -122,9 +128,9 @@ def _load_label(self, idx):
             ymax = (float(xml_box.find('ymax').text) - 1)
             try:
                 self._validate_label(xmin, ymin, xmax, ymax, width, height)
+                label.append([xmin, ymin, xmax, ymax, cls_id, difficult])
             except AssertionError as e:
-                raise RuntimeError("Invalid label at {}, {}".format(anno_path, e))
-            label.append([xmin, ymin, xmax, ymax, cls_id, difficult])
+                logging.warning("Invalid label at %s, %s", anno_path, e)
```

Comment:
is it dangerous to allow invalid bbox in dataset?

Reply:
invalid bboxes are skipped here :)